apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -148,6 +148,19 @@ statements(pgsql) ->
       <<"UPDATE groups SET last_updated_by= $1, updated_at= $2, name= $3"
         "WHERE id= $4">>},
      {delete_group_by_id, <<"DELETE FROM groups WHERE id= $1">>},
+     {list_policies_for_org, <<"SELECT name, policy_group FROM policies WHERE org_id= $1">>},
```

Comment:
Agreed, this is something we are likely to refactor. The goal for this chunk of work is to get something working so we have basic validation, database code, etc. but we will update the API to make it easier to query associations between different objects (e.g., what nodes use which "revision" of a policy, what nodes are in which group), make it more natural to list all objects of a given type (e.g., what groups exist?) and so on. I have a Chef RFC in progress for this which will likely entail some changes to both ChefDK and the server API (and may imply changes in the schema). But for now this gives us a lot of progress towards the end goal.